docs/content/python/getting-started/application/filter-data.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Filter collections of data
+description: |
+  Many applications perform authorization over large collections of data
+  that cannot be loaded into memory. Often index pages showing users a
+  number of resources, like the repositories they can access, will need to
+  use data filtering. The data filtering API provides support for these
+  use cases.
+weight: 4
+---
+
+# Filter collections of data
+
+Many applications perform authorization over large collections of data
+that cannot be loaded into memory. Often index pages showing users a
+number of resources, like the repositories they can access, will need to
+use data filtering. The data filtering API provides support for these
+use cases.
+
+## Authorizing queries
+
+In [Add authorization enforcement](enforce) we discussed enforcing
+*resource-level authorization*. The `authorize` method tells you whether
+*a specific resource* is authorized. But what about endpoints that need
```

Comment:
Slight preference to the rhetorical would be to say:

"But to fetch _all_ authorized resources, we need to use the aptly-named `authorized_resources` instead".